### PR TITLE
DevWs for Focalcrest containing containing ZigBee Switch

### PR DESCRIPTION
**Pull request by Focalcrest-Madi**

### CODE REVIEW

**Review by MadiTang — CHANGES_REQUESTED** [review]
Fix: Add Evvr manufacturer information

**Comment by Focalcrest-Madi** [comment]
Fix: Add Evvr manufacturer information

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -32,18 +32,11 @@ metadata {
...
 		fingerprint profileId: "0104", inClusters: "0000,0003,0004,00005,0006", outClusters: "0000", manufacturer: "LELLKI", model: "JZ-ZB-001", deviceJoinName: "LELLKI Switch"
 		
-		// eZEX 1st Generation Switch
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z0-HA", deviceJoinName: "eZEX Switch"
-		// eZEX 2nd Generation Switch
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z1-HA", deviceJoinName: "eZEX Switch"
-		// eZEX 3rd Generation Switch
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z2-HA", deviceJoinName: "eZEX Switch"
+		// EZEX
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z0-HA", deviceJoinName: "eZEX Switch" //EZEX Switch
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: You shouldn't change/remove other fingerprints. Please make sure to add Your changes on an updated branch.

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 2]
```diff
@@ -80,10 +73,6 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B05", outClusters: "0019", manufacturer: "Leviton", model: "DG15S", deviceJoinName: "Leviton Switch" //Leviton Lumina RF Switch
 		fingerprint manufacturer: "Leviton", model: "DG15A", deviceJoinName: "Leviton Outlet", ocfDeviceType: "oic.d.smartplug" //Leviton Zigbee Plug-In Switch DG15A, Raw Description: 01 0104 010A 00 06 0000 0003 0004 0005 0006 0B05 01 0019
 
-		// NodOn
-		fingerprint profileId: "0104", deviceId: "0002", inClusters: "0000, 0003, 0004, 0005, 0006, 0007, 1000, FC57", outClusters: "0019", manufacturer: "NodOn", model: "SIN-4-1-20", deviceJoinName: "NodOn Switch"
-		fingerprint profileId: "0104", deviceId: "0002", inClusters: "0000, 0003, 0004, 0005, 0006, 0007, 1000, FC57", outClusters: "0019", manufacturer: "NodOn", model: "SIN-4-1-20_PRO", deviceJoinName: "NodOn Switch"
-
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: You shouldn't change/remove other fingerprints. Please make sure to add Your changes on an updated branch.

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 3]
```diff
@@ -105,7 +94,6 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "1000", manufacturer: "SONOFF", model: "01MINIZB", deviceJoinName: "SONOFF 01MINIZB" //01MINIZB
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, FC57", outClusters: "0019", manufacturer: "SONOFF", model: "S26R2ZB", deviceJoinName: "SONOFF Plug", ocfDeviceType: "oic.d.smartplug" //SONOFF S26R2 Plug
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, FC57", outClusters: "0019", manufacturer: "SONOFF", model: "S40LITE", deviceJoinName: "SONOFF Plug", ocfDeviceType: "oic.d.smartplug" //SONOFF S40Lite Plug
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0020, FC57", outClusters: "0019", manufacturer: "SONOFF", model: "ZBMINI-L", deviceJoinName: "SONOFF Switch" //SONOFF ZBMINI-L
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: You shouldn't change/remove other fingerprints. Please make sure to add Your changes on an updated branch.

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 4]
```diff
@@ -32,18 +32,11 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink Outlet", ocfDeviceType: "oic.d.smartplug" //eWeLink SmartPlug (SA-003)
 		fingerprint profileId: "0104", inClusters: "0000,0003,0004,00005,0006", outClusters: "0000", manufacturer: "eWeLink", model: "ZB-SW01", deviceJoinName: "eWeLink Switch"
 		
-		// Minoston
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "Minoston", model: "ZB36S", deviceJoinName: "Minoston Outlet", ocfDeviceType: "oic.d.smartplug" //Minoston SmartPlug
-
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: You shouldn't change/remove other fingerprints. Please make sure to add Your changes on an updated branch.

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Review by KKlimczukS — APPROVED** [review]

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 5]
```diff
@@ -133,6 +133,10 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", outClusters: "0019", manufacturer: "Focalcrest", model: "SRB01", deviceJoinName: "Focalcrest Switch" // In-Wall Relay Switch
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0019", manufacturer: "EVVR", model: "SRB01A", deviceJoinName: "Evvr Switch" // Evvr IRS
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0019", manufacturer: "EVVR", model: "SRB02A", deviceJoinName: "Evvr Switch" // Evvr IRS Lite
+		// Evvr
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy:137]: Just add an empty line above and this one will be ready to go.

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 6]
```diff
@@ -133,6 +133,11 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", outClusters: "0019", manufacturer: "Focalcrest", model: "SRB01", deviceJoinName: "Focalcrest Switch" // In-Wall Relay Switch
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0019", manufacturer: "EVVR", model: "SRB01A", deviceJoinName: "Evvr Switch" // Evvr IRS
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0019", manufacturer: "EVVR", model: "SRB02A", deviceJoinName: "Evvr Switch" // Evvr IRS Lite
+		
+        // Evvr
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: nit: spacing

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Review by KKlimczukS — APPROVED** [review]

**Pull request merged**